incubator/hnc/pkg/kubectl/configset.go

```diff
@@ -0,0 +1,69 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubectl
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	api "github.com/kubernetes-sigs/multi-tenancy/incubator/hnc/api/v1alpha1"
+)
+
+var setTypeCmd = &cobra.Command{
+	Use: fmt.Sprintf("set-type --apiVersion X --kind Y <%s|%s|%s>",
+		api.Propagate, api.Remove, api.Ignore),
+	Short: "Sets the HNC configuration of a specific type",
```

Comment:
`type` in this context isn't that obvious IMO. Maybe `resources type` is better? Thoughts?

Reply:
Hi Ryan! 

Thanks for the suggestion! I feel `set-resources-type` might be a little bit long (e.g., the whole command will be like `kubectl hns config set-resources-type [flags] propagate`) but I don't have a better suggestion about the naming. Are there other kinds of "types" that users might be confused with "resource types"? 

I don't have preference between `type` and `resources-type`. If you think `resources-type` is not too long, we can go for it. :) 

@adrianludwin  WDYT?

Reply:
I was referring just to the  short description, not the actual command names. I agree it'd be too long for the name.

Reply:
Cool! Thanks for the clarification. Changed to "resources type" in the short description.

Reply:
This lgtm!